Horde3D/Samples/Tessellator/CMakeLists.txt

```diff
@@ -60,7 +60,18 @@ elseif( ${CMAKE_SYSTEM_NAME} MATCHES "iOS" )
                                     "-framework CoreMotion" "-framework CoreVideo" "-framework Foundation"
                                     "-framework GameController" "-framework QuartzCore" "-framework UiKit"
                                     "-framework Metal" "-framework OpenGLES")
-    
+
+elseif( ${CMAKE_SYSTEM_NAME} MATCHES "Emscripten" )
```

Comment:
Does tessellator sample work? It uses tessellation that is ES 3.2 functionality that (I might be wrong) should not be available in WebGL2...